.github/workflows/tests.yml

```
@@ -72,4 +72,9 @@ jobs:
 
     - name: Tests
       run: |
-        pytest tests --ignore tests/integration_tests
+        if [ ${{ matrix.python-version }} = 3.9 ]; then
+          pytest tests --ignore tests/integration_tests \
+            --deselect tests/samplers_tests/test_samplers.py::test_nan_objective_value --deselect tests/samplers_tests/test_samplers.py::test_partial_fixed_sampling
```

This line seems to be long. Could you split it?
And also, I think these test cases should be included in the future. So, could you add todo comment to this line?
```suggestion
            --deselect tests/samplers_tests/test_samplers.py::test_nan_objective_value \
            --deselect tests/samplers_tests/test_samplers.py::test_partial_fixed_sampling
```

---

Made the changes in the latest commit 👍